open_seq2seq/data/speech2text/speech_utils.py

```diff
@@ -317,7 +385,9 @@ def get_speech_features_librosa(signal, sample_freq, num_features,
                                 num_fft=None,
                                 dither=0.0,
                                 norm_per_feature=False,
-                                mel_basis=None):
+                                mel_basis=None,
+                                custom_noise=None,
```

Comment:
It looks that right now it makes sense to move `custom_noise` in `augmentation` dictionary and add separate probs for each type of data augmentation there as well (instead of `prob_noise` argument).